odl/operator/fn_ops.py

```diff
@@ -0,0 +1,406 @@
+## Copyright 2014-2016 The ODL development group
+##
+## This file is part of ODL.
+##
+## ODL is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## ODL is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+#
+#"""Default operators defined on any (reasonable) space."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+from odl.operator.operator import Operator
+from odl.space import fn
+from odl.set import LinearSpace, Field
+#from odl.util import writable_array
+
+import numpy as np
+
+__all__ = ('SamplingOperator', 'ZerofillingOperator', 
+           'VectoriseOperator', 'EmbeddingOperator')
+
+class SamplingOperator(Operator):
+
+    """Operator that samples coefficients.
+
+        ``SamplingOperator(f)(i) == cell_volume * f(i) \approx \int_cell f(x) dx``
+    """
+
+    def __init__(self, domain, indices):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        domain : `LinearSpace` or `Field`
+            Set of elements on which this operator acts.
+        indices : list
+            List of indices where to sample the function
+
+        Examples
+        --------
+        >>> X = odl.uniform_discr(min_pt=[0, 0], max_pt=[1, 1], shape=[2, 2])
+        >>> x = X.element(range(X.size))
+        >>> ind = list()
+        >>> ind.append(range(2))
+        >>> ind.append([1,1])
+        >>> A = odl.operator.SamplingOperator(X, ind)
```

Comment:
If you add `fn_ops` [here](https://github.com/odlgroup/odl/blob/master/odl/__init__.py), you could write this as `odl.SamplingOperator`.